system_metrics_collector/src/system_metrics_collector/periodic_measurement_node.cpp

```diff
@@ -33,40 +33,40 @@ PeriodicMeasurementNode::PeriodicMeasurementNode(
   const std::chrono::milliseconds publish_period,
   const bool clear_measurements_on_publish)
 : Node(name),
-  measurement_period_(measurement_period),
   publishing_topic_(publishing_topic),
-  measurement_timer_(nullptr),
-  publish_timer_(nullptr),
+  measurement_period_(measurement_period),
   publish_period_(publish_period),
-  clear_measurements_on_publish_(clear_measurements_on_publish)
-{}
+  clear_measurements_on_publish_(clear_measurements_on_publish) {}
 
 bool PeriodicMeasurementNode::setupStart()
 {
+  if (publisher_ == nullptr) {
+    publisher_ = create_publisher<MetricsMessage>(publishing_topic_, 10);
```

Comment:
Why 10? For numerical arguments, please use `create_publisher<MetricsMessage>(publishing_topic_, 10 /* parameter name */);` or use a named constant.

Reply:
Added comment.